docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -3,15 +3,38 @@
 
 [role="_abstract"]
 
-A devfile describes and defines a cloud native development environment.
+Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development while providing a way for being customized or extended to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating what was previously done manually.
```

Comment:
At some RH writing training we were advised to write for "non-native speakers with IELTS level 3" (out of 9). While the meaning of the 2nd sentence seems to be clear, do you think grammar here is on the more advanced side and can cause some readers troubles?

E.g. this part: "while providing a way for being customized or extended to fit". If you decide to rephrase it, possible ways could be "users/you can customize and extend devfiles to..." or "... and can be customized or extended to fit..." (but it's passive voice, it's frowned upon in general).

Reply:
Also, this part might also be tricky, WDYT? But not too much, it's up to you, of course, if you'd like to keep it.
   "devfiles provide instructions for automating what was previously done manually"
Will this version be a bit more lightweight?
   "for automating manual processes"